rust/template/differential_datalog/src/program/mod.rs

```diff
@@ -988,70 +991,49 @@ enum Reply {
 }
 
 impl Program {
-    /// Instantiate the program with `nworkers` timely threads.
+    /// Instantiate the program with `number_workers` timely threads.
     pub fn run(&self, number_workers: usize) -> Result<RunningProgram, String> {
+        let config = Config {
+            num_timely_workers: number_workers,
+            ..Default::default()
```

Comment:
I might be missing something, but I think this will set profiling mode to `None`, so neither self-profiling, which is currently always enabled by default, nor native timely/differential profiling will work. Since `run_with_config` is currently not callable through the DDlog API, there is basically no way for users to enable any kind of profiling.

Reply:
This is my only remaining concern (well, this and the failing pipeline).  Can we address those, merge, and throw a party?

Reply:
Set the profiling mode to `SelfProfiling` in `Program::run`, now waiting on CI